test/testproject-go/documentformat.go

```diff
@@ -0,0 +1,4 @@
+package main
+func main() {
+print("hello, world!")
```

Comment:
it will be good if we can have mutli byte chars.

Reply:
alpha/beta are universal and easy for anyone coming from different language to speak.

https://github.com/prabirshrestha/vim-lsp/blob/master/test/testfiles/multibyte.txt

Reply:
Thank you for the missing point. I added multibyte texts in the test case at b81b79a.